examples/multi-functions-synchronization/ai_ao_sync.py

```diff
@@ -0,0 +1,98 @@
+﻿"""Example of analog input and output synchronization.
+
+This example demonstrates how to continuously acquire and
+generate data at the same time, synchronized with one another.
+"""
+
+from typing import Tuple
+
+import numpy as np
+import numpy.typing
+
+import nidaqmx
+from nidaqmx.constants import AcquisitionType
+
+
+def generate_sine_wave(
+    frequency: float,
+    amplitude: float,
+    sampling_rate: float,
+    phase_in: float,
+    number_of_samples: int,
+) -> Tuple[numpy.typing.NDArray[numpy.double], float]:
```

Comment:
As per my last comment on the AO review:
- Swap phase_in and number_of_samples
- Default phase_in: float = 0.0
- Remove phase from main()
- Use `ao_data, _ = generate_sine_wave(...)`